packages/ndla-icons/src/action/FourlineHamburger.tsx

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2021-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+// N.B! AUTOGENERATED FILE. DO NOT EDIT
+import React, { ReactNode } from 'react';
+import Icon, { Props as IconProps } from '../Icon';
+
+interface Props extends IconProps {
+  children?: ReactNode;
+}
+
+const FourlineHamburger = (props: Props) => (
+  <Icon viewBox="0 0 24 24" data-license="Apache License 2.0" data-source="Material Design" {...props}>
```

Comment:
Er alle nye ikon faktisk henta fra sida det er lenka til i designmanualen? Om det er noen som er laga nye så ta for all del credits! Vi har allerede ikoner med source "SopraSteria"